components/brave_wallet_ui/common/constants/local-storage-keys.ts

```diff
@@ -1,11 +1,7 @@
-// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// Copyright (c) 2018 The Brave Authors. All rights reserved.
 // This Source Code Form is subject to the terms of the Mozilla Public
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
-// you can obtain one at http://mozilla.org/MPL/2.0/.
-
-/**
- * Key for localStorage object that records if the user has closed the sweepstakes banner.
```

Comment:
Created https://github.com/brave/brave-core/pull/15870 to revert it. This seems to be the only file which was affected.

Interestingly, I'm not sure that that comment is correct because the same comment is also here, on a different key (which looks more correct).
https://github.dev/brave/brave-core/blob/d16bacac881246605459178e13f82b90c6471def/components/brave_wallet_ui/components/desktop/sweepstakes-banner/sweepstakes-banner.tsx#L31

Do you think it's worth  reverting the change (for consistencies sake)?

Reply:
> This seems to be the only file which was affected.
I was worried if there were other unnecessary cleanups.

> I'm not sure that that comment is correct
yeah, the comment looks not perfect. I guess the wallet team may decide to correct it.